### ISSUE TITLE
[Spell 46584] Raise dead not working

### ISSUE THREAD

**Issue by iliektetris** [cmangos/wotlk-db#189]
When clicked, nothing happens, spell goes into infinite cooldown as well.
46585 and 52150 work, seems its not a db issue?

**Comment by killerwife** [comment]
Likely a missing script to clear infinite cooldown is missing on death of 46585 summon. Thats what you should try to implement.

**Comment by iliektetris** [comment]
Aside from cooldown not working, summoning also doesn't work.
46584 does nothing, 46585 and 52150 summon ghouls just fine.
So I guess a script is needed for 46584 to use 46585 when you dont have Master of Ghouls talent and use 52150 when talent is unlocked?

**Comment by killerwife** [comment]
Yes, all of that needs to be added. Sadly I am not a wotlk dev, I just know the general logic how it should work.

**Comment by pillbug22** [comment]
@iliektetris when you say you successfully cast the individual spell IDs, was it with gm commands (and the pet appeared)?

I have the same issue where using the built-in icon for Raise Dead goes on infinite CD, and no pet appears. Trying /cast Raise Dead with normal player macro commands has the same effect.

Looking in the database, the only SpelI ID I see with Corpse Dust as a reagent is ID# 48289. I've tried using that in-game with .cast 48289, but nothing happens (no pet, no CD triggered).

**Comment by al3xc1985** [comment]
@killerwife I tested this, and found that is working as intended. No more infinite CD
